SoSe16/Bericht_SoSe16_Konstanz.tex

```diff
@@ -20,13 +20,16 @@
 \definecolor{urlred}{HTML}{660000}
 \usepackage{hyperref}
 \hypersetup{
-      colorlinks=true,
-      linkcolor=black,                              % Farbe der internen Links (u.a. Table of Contents)
-      urlcolor=urlred}                              % Farbe der url-links
+	colorlinks=true,
+	linkcolor=black,                              % Farbe der internen Links 
+	%(u.a. Table of Contents)
+	urlcolor=urlred}                              % Farbe der url-links
 \parindent 0pt                                      % Absatzeinrücken verhindern
-\parskip 12pt                                       % Absätze durch Lücke trennen
+\parskip 12pt                                       % Absätze durch Lücke 
+%trennen
```

Comment:
Das ist mein Fehler. Ich hab Lux ein fixe 80 column rule eingebaut. Sein Editor
ist da leider etwas unintelligent mit...

On Mon, Jun 20, 2016 at 08:38:46AM -0700, fabianfreyer wrote:
> >  \parindent 0pt                                      % Absatzeinrücken verhindern
> > -\parskip 12pt                                       % Absätze durch Lücke trennen
> > +\parskip 12pt                                       % Absätze durch Lücke
> > +%trennen
>
> Warum ist das auf einer neuen Zeile?
>
> ---
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/ZaPF/ZaPF_Berichte/pull/6/files/9cba85cfcbebfb234d0e9fc4a6ae667752357b1a#r67712149